project/kediatr-core/src/test/kotlin/com/trendyol/kediatr/PipelineBehaviorTest.kt

```diff
@@ -7,65 +7,99 @@ import org.junit.jupiter.api.assertThrows
 import kotlin.test.assertEquals
 import kotlin.test.assertTrue
 
-var asyncPipelinePreProcessCounter = 0
-var asyncPipelinePostProcessCounter = 0
-var pipelinePreProcessCounter = 0
-var pipelinePostProcessCounter = 0
-var pipelineExceptionCounter = 0
-var asyncPipelineExceptionCounter = 0
+var asyncExceptionPipelineBehaviorHandleCounter = 0
```

Comment:
There is no need to keep the `async` prefix since all the code is using `suspend` anymore.

Reply:
Removed the `async` prefix to all counter variables with last commit.